### PR TITLE
cli: Add LB IP to cilium status

### PR DESCRIPTION
**Pull request by brb**
This commit extends cilium status to include NodePort BPF IP addr (LB
IP) for each device which has bpf_host.o attached (the program is
doing N-S loadbalancing).

The example output:
```
$ cilium status
[..]
KubeProxyReplacement:   Strict   [wlp3s0 172.20.10.2 (Direct Routing)]
```

### CODE REVIEW

**Review by borkmann — APPROVED** [review]

**Comment by brb** [comment]
test-me-please

File: api/v1/openapi.yaml [hunk 1]
```diff
@@ -1754,9 +1754,28 @@ definitions:
...
         - Partial
       devices:
-          type: array
-          items:
-            type: string
+        description: |-
+
+
+          +k8s:deepcopy-gen=true
+        type: array
```
Comment (christarazi) [inline api/v1/openapi.yaml:1761]: Was there a reason for these extra whitespaces? We should followup with some documentation on our API fields.

Comment (brb) [reply api/v1/openapi.yaml:1761]: > Was there a reason for these extra whitespaces?

@aanm might have an answer.

> We should followup with some documentation on our API fields

IMHO, the added fields are self-explanatory.

Comment (aanm) [reply api/v1/openapi.yaml:1761]: It's to avoid having the documentation of this fields starting with `+k8s:deepcopy-gen=true`

**Pull request merged**